CHANGELOG.md

```diff
@@ -5,6 +5,32 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [v3.74.0](https://github.com/buildkite/agent/tree/v3.73.1+1) (2024-06-06)
+[Full Changelog](https://github.com/buildkite/agent/compare/v3.73.1...v3.74.0)
+
+### Added
+- kubernetes-exec is now a flag [#2814](https://github.com/buildkite/agent/pull/2814) (@DrJosh9000)
```

Comment:
This only affects agents that run under the k8s stack - I think it's more reasonable to group this with internal. It's definitely more than "Added".